packages/web-app-preview/src/App.vue

```diff
@@ -89,6 +93,18 @@
         >
           <oc-icon size="large" name="arrow-drop-right" />
         </oc-button>
+        <div class="oc-flex">
+          <oc-button
+            v-oc-tooltip="toggleFullScreenDescription"
+            class="preview-controls-fullscreen"
+            appearance="raw"
+            variation="inverse"
+            :aria-label="toggleFullScreenDescription"
+            @click="isFullScreenModeActivated = !isFullScreenModeActivated"
+          >
+            <oc-icon fill-type="line" name="fullscreen" />
```

Comment:
Could you try using the icon `fullscreen-exit` when the fullscreen mode is active?